modules/component/src/componentNext.ts

```diff
@@ -170,6 +201,38 @@ export class ComponentNext<P1 extends ComponentProps> {
     )
   }
 
+  /**
+   * Adds child component to a component, this operator does multiple things i.e
```

Comment:
in addition to reformatting for imperative voice, could be made clearer:
- Add component as a child of a given component